### PR TITLE
#13 Added Docker-Compose

### PR DESCRIPTION
**Pull request by mtracewicz**
I have added docker-compose.
For this to work I have added a Dockerfile and .dockerignore for the frontend.
I have also disabled CQRS for our API (this might be changed if we decide that API should not be open).
There is a warning from linter as I have added a `console.log` which demonstrates that integration works properly when a user does `docker-compose up`

### CODE REVIEW

File: Docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+version: "3.9"
+services:
+  frontend:
```
Comment (Sojusan) [inline Docker-compose.yml:3]: I think it would be nice to have [Healthcheck](https://docs.docker.com/compose/compose-file/compose-file-v3/#healthcheck) on our frontend and backend services. Healthcheck is an action to check if our service is still running and if it is still responding. For example, our backend app could have an endpoint to return the current version and the healthcheck will be sending requests on that endpoint and that will ensure us that our app is working correctly.

File: Docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+version: "3.9"
```
Comment (Sojusan) [inline Docker-compose.yml:1]: I think this file should have filename written in lowercase as `docker-compose.yml` is a default value in [official documentation](https://docs.docker.com/compose/compose-file/compose-file-v3/#service-configuration-reference).

**Review by Sojusan — CHANGES_REQUESTED** [review]

File: Docker-compose.yml (hunk 2 above)
Comment (mtracewicz) [reply Docker-compose.yml]: Done 😄

File: Docker-compose.yml (hunk 1 above)
Comment (mtracewicz) [reply Docker-compose.yml]: While I agree that health checks can be useful I believe that We don't need to add them now. 
In my opinion, frontend does not need a health check at all as the Docker image is used only for development/debugging purposes as it will most likely be deployed to GitHub pages. As for the backend, it might be useful however again the purpose of this docker-compose is for development/debugging and not deployment.  We would need to use some service for deployment which monitors this (like Azure Web Apps) which will be much more expensive than a cheap VPS.

So to sum it up I would say We might want to add a health check later on for the backend. But I would skip this, for now, to start working on an actual app code. 

What do You think?

Comment (Sojusan) [reply Docker-compose.yml]: Ok, I see Your point. I agree, this could be added anytime. Let's focus on the app code.

**Review by Sojusan — APPROVED** [review]
